modules/rosa-create-objects.adoc

```diff
@@ -116,6 +116,9 @@ $ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 |--private
 |Restricts master API endpoint and application routes to direct, private connectivity.
 
+|--private-link
+| Specifies to use AWS PrivateLink to provide private connectivity between VPCs and services.
```

Comment:
Should we call out that `--subnet-ids` is a requirement when using `--private-link`?

Reply:
@wgordon17  Yes, we should call out the `--subnets-ids` as a requirement when using `--private-link`. It should be mentioned with this `--private-link` entry. It can also be added as a requirement under `--subnet-ids` as an additional paragraph.